tstest/integration/integration_test.go

```diff
@@ -493,6 +493,7 @@ func (n *testNode) StartDaemon(t testing.TB) *Daemon {
 		"TS_LOG_TARGET="+n.env.LogCatcherServer.URL,
 		"HTTP_PROXY="+n.env.TrafficTrapServer.URL,
 		"HTTPS_PROXY="+n.env.TrafficTrapServer.URL,
+		"TS_NETNS_BIND_IFC=lo",
```

Comment:
I really dislike this sort of thing. I don't want to be holding the integration test's hand. I fear this will bite us in both directions in the future: break things it shouldn't (binding to lo when we don't want that), and hide broken routing in tailscaled by us telling it what to do.

The netns stuff is used by:

* magicsock UDP conns
* DERP TCP dials
* control TCP dials

Which one(s) are the problem?

If it's just magicsock, can we instead have magicsock use a different socket for localhost peers? We've been wanting to do that for a long time anyway (go multi-socket) and this would be a nice baby step into that.